src/Fabulous.Avalonia/Views/Controls/NumericUpDown.fs

```diff
@@ -77,6 +98,14 @@ module NumericUpDownBuilders =
                 NumericUpDown.ValueChanged.WithValue(ValueEventData.create value (fun args -> valueChanged args |> box))
             )
 
+        static member inline NumericUpDown<'msg>(min: decimal, max: decimal, value: decimal option, valueChanged: decimal option -> 'msg) =
```

Comment:
I would recommend you use `float` instead of `decimal` to be consistent with the other widgets